iceoryx_examples/user_header/sleep_for.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
```

Comment:
This is now the fourth sleep_for header with the same function. Maybe we should think of that we make this a common header that is accessible by all examples to reduce code duplication. This has the disadvantage that the examples are not so self-contained anymore.

Reply:
Maybe instead of adding this to the example we can move it to utils/hoofs?

Reply:
this is C code and should not be in the hoofs. Albeit, if the oryx gets a horseshoe, it would look like a C :laughing: ... maybe we got the logo for the hoofs, a horseshoe with two `+` or two `x`